AgileBoard.Api/src/AgileBoard.Api/Controllers/V1/UserStoriesController.cs

```diff
@@ -0,0 +1,115 @@
+﻿using AgileBoard.Api.Contracts.Requests;
+using AgileBoard.Api.Contracts.Responses;
+using AgileBoard.Api.Contracts.V1;
+using AgileBoard.Api.Domain;
+using AgileBoard.Api.Extensions;
+using AgileBoard.Api.Services;
+using AutoMapper;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AgileBoard.Api.Controllers.V1
+{
+    [ApiController]
+    [Route("[controller]")]
```

Comment:
it's better to define version on top of controller, so you would have api/v1/agile-board/

Reply:
so there should not be any controller name? 
![image](https://user-images.githubusercontent.com/20928680/179097024-f88a6464-dd31-441e-8aab-72a3d6a838ee.png)

Reply:
No, it's okay but you keep version and rest of that in each endpoint instead of controller :)